src/containers/Info/Info.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+
+import { Header, H1, H2, HeaderParagraph, HrefHeader, Markdown } from '../../components';
+
+import { Info, Server } from '../../common';
+
+import ServersComponent from '../Servers/Servers';
+import ContactComponent from './Contact';
+import LicenseComponent from './License';
+
+import { Info as InfoWrapper, InfoHeader } from './styled';
+
+export interface InfoProps {
+  info: Info;
+  servers?: Server[];
+  showServers: boolean;
+}
+
+class InfoComponent extends Component<InfoProps> {
+  public render() {
+    const { info: { title, version, description, termsOfService, contact, license }, servers, showServers } = this.props;
+
+    return (
+      <InfoWrapper>
+        <InfoHeader>
+          <H1>{title} {version}</H1>
+          {description ? <Markdown>{description}</Markdown> : null}
```

Comment:
Please replace ternary operator with guard whenever you return null when a condition is not met